bin/bundle

```diff
@@ -1,104 +1,95 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
+
 #
 # This file was generated by Bundler.
 #
 # The application 'bundle' is installed as part of a gem, and
 # this file is here to facilitate running it.
 #
 
```

Comment:
- [ ] It would be nice if you use the latest template of microverse readme file because I can see that you using the old template of readme 😨 Kindly use the latest template to follow the project requirements 👍